api/edge/v1alpha1/types.go

```diff
@@ -205,8 +205,9 @@ type PlacementDecisionSpec struct {
 }
 
 // DownsyncObjectReferences explicitly defines the objects to be down-synced.
-// The ClusterScope list defines the cluster-scope objects, while NamespacedObjects packs individual objects
-// identifiable by namespace & name.
+// The ClusterScope list defines the cluster-scope objects, NamespacedObjects packs individual objects
+// identifiable by namespace & name, and generation represents the internal spec generation of the listed
```

Comment:
Instead of "NamespacedObjects packs individual objects identifiable by namespace & name" why not just do the complement to the cluster-scoped clause --- "NamespacedObjects list defines the namespaced objects"?

Reply:
right. this comment was left from legacy code (since we took structure from syncerconfig).
will change this.

Reply:
fixed